common/src/main/webapp/less/style.less

```diff
@@ -654,7 +654,7 @@ tfoot {
   margin: 0 0;
   background-color: var(--obs-badge-background);
   border: solid 1px var(--site-border-color);
-  font-size: inherit;
+  font-size: smaller;
```

Comment:
This is unrelated. It's a fix to a previous PR which resulting in showing text too big in badges.